xAPI.md

```diff
@@ -304,17 +304,33 @@ be unintuitive and/or lengthy to dissect into a list of requirements.
 
 ## 3.0 Guidenelines for Implementing Technical Specificationss Like xAPI
 
-## 4.0 Binding to JavaScript Object Notation (JSON)
+## 4.0 Serialization and JavaScript Object Notation (JSON)
 
-### 4.1 Serialization
+Serialization is the process of translating data objects and structures into a format for storage or transmission, such that the original data object can 
+be recreated from the resulting serialization. In some cases it might be possible to serialize a piece of data in more than one way, for example a boolean 
+property with a value of true might be represented as ```true``` or ```1``` depending on the serialization used. 
 
-### 4.2 Population of JSON Properties
+xAPI follows the rules of JSON for serializations (so boolean values are represented as ```true``` or ```false```). It might also be possible to
+represent the objects defined in this specification using other serializations, such as XML. This is out of scope of this 
+specification and use of anything other than JSON to represent the objects defined in this specification is not conformant
+with this specification. 
+
+Even within the rules of JSON there are
+possible variations of how data can be serialized, especially in relation to data about time. This is significant as a number of features of xAPI 
+rely on systems being able to determine whether two Statements are equivalent. See <a href="#statement-immutability-and-exceptions"> Immutability 
+and exceptions</a> for more details about the properties of the Statement affected by this. 
+
+JSON allows for obejcts to have properties that do not contain values. This is not recommended within xAPI; if the statement is not intended to contain
+data about a property then it is expected that the property will not be used at all. All required properties are required to contain values. 
+
+###### Requirements
+* Statements and other objects SHOULD NOT include properties without values. 
```

Comment:
Can we clarify what we mean by empty and null? 
```{"foo": null}``` is what I think we're talking about.

```{"foo":}``` and ```{"foo"}``` are not valid json. 

Or does empty mean ```{"foo": ""}```? There are some very valid reasons why an empty string would be the value of some properties e.g. the result.response.